index.html

```diff
@@ -114,7 +114,36 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
-
+<!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+<div class="card">
+  <p class="name">Bob</p>
+  <p class="contact">
+    <i class="fab fa-twitter"></i>
+    <a href="https://twitter.com/ninjaChickenplz" target="_blank">Me</a>
+  </p>
+  <p class="about">about you</p>
+  <div class="resources">
+    <p>3 Useful Dev Resources</p>
+    <ul>
+      <li>
+        <a href="https://www.reddit.com/r/ProgrammerHumor/" target="_blank" title="First Resource">
+          Just For laugh
+        </a>
+      </li>
+      <li>
+        <a href="https://www.youtube.com" target="_blank" title="Second Resource">
+         Youtube
+        </a>
+      </li>
+      <li>
+        <a href="https://www.reddit.com/r/programming/" target="_blank" title="Third resource">
+          Reddit Programming
+        </a>
+      </li>
+    </ul>
+  </div>
+</div>
+<!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
Please change this comment as well. 
`<!-- ________*END TEMPLATE* Contributor card END ________ -->` 
to `<!-- ________ Your_Name Contributor card END ________ -->`